.env.example

```
@@ -1,13 +1,23 @@
-# Copy to .env — the server loads it on boot.
-
 PORT=3400
-JWT_SECRET=
-DATABASE_URL=
-STORAGE_DIR=./uploads
+JWT_SECRET=verysecrettoken
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the PORT key

(UnorderedKey)